src/Language/PureScript/ModuleDependencies.hs

```diff
@@ -62,5 +62,5 @@ toModule (AcyclicSCC m) = return m
 toModule (CyclicSCC [m]) = return m
 toModule (CyclicSCC ms) =
   throwError
-    . errorMessage' (getModuleSourceSpan (head ms))
+    . errorMessage'' (NEL.fromList (fmap getModuleSourceSpan ms))
```

Comment:
Could we avoid the partial NEL.fromList and just pattern match the head out of `ms`?

Reply:
I mean, we can... I just figured since we can easily see ms is not empty from the pattern already being covered that it probably wasn't necessary?

Reply:
I mean we'd just move the partiality into the top level pattern match as that wouldn't match the empty list anymore, but I think fromList is one of those unexpectedly partial functions like maximum...